DESCRIPTION

```diff
@@ -1,14 +1,20 @@
 Package: nanotime
 Type: Package
 Title: Nanosecond-Resolution Time for R
-Version: 0.2.4.2
-Date: 2019-11-21
+Version: 0.2.4.3
+Date: 2019-11-28
 Author: Dirk Eddelbuettel and Leonardo Silvestri
 Maintainer: Dirk Eddelbuettel <edd@debian.org>
 Description: Full 64-bit resolution date and time support with resolution up
  to nanosecond granularity is provided, with easy transition to and from the
  standard 'POSIXct' type.
-Imports: methods, bit64, RcppCCTZ (>= 0.2.3), zoo
+Imports: methods, bit64, RcppCCTZ (>= 0.2.6.3), zoo
 Suggests: tinytest, data.table, xts
+LinkingTo: Rcpp, RcppCCTZ
 License: GPL (>= 2)
-RoxygenNote: 7.0.0
+RoxygenNote: 7.0.2
+Collate:
```

Comment:
Ack. I never use / used / needed that roxygen feature (yet). 

So I guess now we have co-depends with R files among S4 classes so the collation order matters?

Reply:
Exactly. Otherwise we get a bunch of warnings when the package loads that we are referring to non-existing classes. And it's more elegant that naming the files `10-...`, `20-...`, etc.

Reply:
Yep. That's the old-old scheme I have seen people use in the past.